dev/workflow/rocoto/gfs_tasks.py

```diff
@@ -2912,6 +2912,11 @@ def earc_tars(self):
         if 'enkfgdas' in self.run:
             dep_dict = {'type': 'metatask', 'name': f'{self.run}_epmn'}
             deps.append(rocoto.add_dependency(dep_dict))
+            if self._base.get('DOHYBVAR_OCN', True):
+                dep_dict = {'type': 'task', 'name': f'gdas_ocnanalecen'}
+                deps.append(rocoto.add_dependency(dep_dict))
+                dep_dict = {'type': 'task', 'name': f'gdas_marineanlfinal'}
+                deps.append(rocoto.add_dependency(dep_dict))
```

Comment:
Are these already dependencies for the ensemble forecasts?  If so, you may not need to add them here, since `enkfgdas_earc` is dependent on the `epos` jobs.